packages/manager/modules/carrier-sip/src/carrier-sip.service.js

```diff
@@ -0,0 +1,63 @@
+import map from 'lodash/map';
+
+import VoipCarrierSipLine from './VoipCarrierSipLine.class';
+
+export default class CarrierSipService {
+  /* @ngInject */
+  constructor($q, OvhApiTelephony) {
+    this.$q = $q;
+    this.OvhApiTelephony = OvhApiTelephony;
+  }
+
+  fetchAll(billingAccount) {
+    return this.OvhApiTelephony.CarrierSip().v6().query({
+      billingAccount,
+    }).$promise.then(carrierSipLines => this.$q.all(carrierSipLines
+      .map(carrierSip => this.getCarrierSip(
```

Comment:
Iceberg instead ?

Reply:
cc @JeremyDec

Reply:
Yes, I'll make the changes ;)

Reply:
@antleblanc / @marie-j => https://github.com/ovh-ux/ovh-api-services/pull/241